platform/osgi/platform-osgi-condpermadmin/src/main/java/org/codice/ddf/condpermadmin/PermissionActivator.java

```diff
@@ -58,6 +58,9 @@ public void start(BundleContext bundleContext) throws Exception {
     permAdminTracker.open();
 
     ConditionalPermissionAdmin conditionalPermissionAdmin = permAdminTracker.getService();
+    if (conditionalPermissionAdmin.getConditionalPermissionInfos().hasMoreElements()) {
```

Comment:
✏️ We may want to add a comment here to explain why we have this conditional and what someone would have to do if they need to update the permissions. If we can link to tickets that will address this and allow us to remove this work-around, that'd be great too.